test/specs/productDetailed.js

```diff
@@ -0,0 +1,28 @@
+const ProductsPage = require("../pageobjects/ProductsPage");
+const LoginPage = require("../pageobjects/LoginPage");
+const ProductDetailedPage = require("../pageobjects/ProductDetailedPage");
+const {CREDENTIALS} = require("../helper/credentials");
+const {ImageSource} = require("../helper/inventoryData");
+
+
+describe(" Product Detailed Page Tests", () => {
+
+    describe("Access the 'Sauce Labs Backpack' details upon clicking on image", () => {
+
+        beforeEach(async function () {
+            await LoginPage.open();
+            await LoginPage.login(CREDENTIALS.standard, CREDENTIALS.password);
+        });
+
+        afterEach(async function () {
+            await ProductsPage.logout();
+        });
+
+        it("should display only the 'Sauce Labs Backpack' details upon clicking on its image", async () => {
+            await ProductsPage.allItemsImages[0].click()
+
+            expect(await ProductDetailedPage.detailsImage.isDisplayed()).toBe(true);
+            expect(await ProductDetailedPage.detailsImage.getAttribute('src')).toEqual(ImageSource.backpack);
```

Comment:
It's just checking for the image. It should also check for item name, description, price, add to cart button.

Reply:
the TCs for the productDetails are still under development. the checks for the item name, description, price and the add to cart button will be added